app/tests/e2e/test_fixtures.py

```diff
@@ -85,7 +86,12 @@ async def credential_exchange_id(
     credential_exchange_id = credential_exchange["credential_id"]
     assert credential_exchange["protocol_version"] == "v1"
 
-    time.sleep(5)
+    assert check_webhook_state(
```

Comment:
Ah I see you added an `topic` option. For completeness record_id would also be good!